diff --git a/Makefile b/Makefile
index 44fb298..d8c4a53 100644
--- a/Makefile
+++ b/Makefile
@@ -154,7 +154,7 @@ ifneq ($(filter armv6%,$(UNAME_M)),)
 endif
 ifneq ($(filter armv7%,$(UNAME_M)),)
 	# Raspberry Pi 4
-	CFLAGS += -mfpu=neon-fp-armv8 -mfp16-format=ieee -mno-unaligned-access -funsafe-math-optimizations
+	CFLAGS += -mfpu=neon-fp-armv8 -mfp16-format=ieee -mno-unaligned-access
 endif
 ifneq ($(filter armv8%,$(UNAME_M)),)
 	# Raspberry Pi 4
@@ -193,7 +193,6 @@ clean:
 
 main: main.cpp ggml.o utils.o
 	$(CXX) $(CXXFLAGS) main.cpp ggml.o utils.o -o main $(LDFLAGS)
-	./main -h
 
 quantize: quantize.cpp ggml.o utils.o
 	$(CXX) $(CXXFLAGS) quantize.cpp ggml.o utils.o -o quantize $(LDFLAGS)
diff --git a/ggml.h b/ggml.h
index bac4fe6..d60520d 100644
--- a/ggml.h
+++ b/ggml.h
@@ -1,5 +1,8 @@
 #pragma once
 
+// so we find clock_gettime etc
+#define _POSIX_C_SOURCE 199309L
+
 //
 // GGML Tensor Library
 //
